component/src/main/java/org/wso2/extension/siddhi/io/wso2event/source/WSO2EventSourceDS.java

```diff
@@ -18,6 +18,8 @@
 
 package org.wso2.extension.siddhi.io.wso2event.source;
 
+import io.siddhi.core.exception.ConnectionUnavailableException;
+import io.siddhi.core.stream.input.source.Source;
```

Comment:
Same as above.
Sort order of import packages matters

Reply:
According to WSO2 checkstyles guidelines[1], these imports should be ordered according to alphabetical order. If we did not follow that a checkstyle error occurs when building the package.

ex:
`/Users/raveen/WSO2/Public/Siddhi-Extensions/siddhi-io-wso2event/component/src/main/java/org/wso2/extension/siddhi/io/wso2event/sink/WSO2EventSink.java:22: error: Wrong order for 'io.siddhi.annotation.Example' import.
`

[1] https://github.com/wso2/code-quality-tools/blob/master/checkstyle/checkstyle.xml#L97